packages/block-library/src/embed/index.js

```diff
@@ -136,6 +137,58 @@ export function getEmbedEdit( title, icon ) {
 			return false;
 		}
 
+		/**
+		 * Sets the appropriate CSS class to enforce an aspect ratio when the embed is resized
+		 * if the HTML has an iframe with width and height set.
+		 *
+		 * @param {string} html The preview HTML that possibly contains an iframe with width and height set.
+		 */
+		maybeSetAspectRatioClassName( html ) {
+			const previewDom = document.createElement( 'div' );
```

Comment:
Minor: DOM is an acronym and as such should be capitalized as `previewDOM` ([reference](https://github.com/WordPress/gutenberg/blob/master/docs/reference/coding-guidelines.md#variable-naming))